redeem/gcodes/M106_M107.py

```diff
@@ -41,7 +44,8 @@ def get_long_description(self):
         return "Set the current fan power. Specify S parameter for the " \
                "power (between 0 and 255) and the P parameter for the fan " \
                "number. P=0 and S=255 by default. If no P, use fan from config. "\
-               "If no fan configured, use fan 0. If 'R' is present, ramp to the value"
+               "If no fan configured, use fan 0. If 'R' is present, ramp to the value"\
+               "if 'F' present change PWM frequency hz eg. F1000 is 1khz"
```

Comment:
thank you for being thorough and updating the description too!

appreciate the contribution : )